### PR TITLE
This is the addition of question 4 and 5

### PR DESCRIPTION
**Pull request by brandont2801**

### CODE REVIEW

File: project_work.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+print("=" * 19)
+
+# data sets for entry into the graphs
+states = list(set(state_list))
+sales = list(state_dict.keys())
+
+
+
+# creates the bar chart for question 2
+fig = plt.figure(1)
```
Comment (gagebroberg) [inline project_work.py:192]: "fig = " can be deleted

File: project_work.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+texas_y_points = amount_sale_month_state(date_state_sale_dict, 'Texas')
+cali_y_points = amount_sale_month_state(date_state_sale_dict, 'California')
+florida_y_points = amount_sale_month_state(date_state_sale_dict, 'Florida')
+ohio_y_points = amount_sale_month_state(date_state_sale_dict, 'Ohio')
+nevada_y_points = amount_sale_month_state(date_state_sale_dict, 'Nevada')
+car_data_file.close()           #All y-points created
+
+plt.xlabel('Months')        #Customization
+plt.ylabel('Amount of Sale')
+plt.title('Amount of Sale in Different Months in Each State')
```
Comment (gagebroberg) [inline project_work.py:242]: These lines need to be under the plt.figure(4) or else they aren't added to the figure

File: project_work.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+plt.figure(4)       #Plots the points with respective legend
```
Comment (gagebroberg) [inline project_work.py]:
```suggestion
plt.figure(4, figsize = (12,5))       #Plots the points with respective legend

```
This will make the figure bigger so that the month ticks are not overlapping

File: project_work.py (hunk 2 above)
Comment (gagebroberg) [reply project_work.py:242]: In fact, these are currently the labels and titles of the pie chart as the code is written currently.

File: project_work.py [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+    sizes.append(brand_percentage[1])
+colors = ['b', 'g', 'r', 'c', 'm', 'y']
+number = 0
+for num in range(len(percentage_brand_dict_sorted)):    #Helps assign colors to all elements of chart
+    if number == len(percentage_brand_dict_sorted) - 1:
+        number = 0
+    colors.append(colors[number])
+    number += 1
+plt.pie(sizes, labels = labels, colors = colors, autopct='%1.2f%%')
+
```
Comment (gagebroberg) [inline project_work.py:217]: The pie chart needs a proper title.

File: project_work.py [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+percentage_brand_dict_sorted = sorted(percentage_brand_dict.items(), key=lambda item: item[1], reverse=True)
+for brand_and_percentage in percentage_brand_dict_sorted:
+    print(f"{brand_and_percentage[0]} : {round(brand_and_percentage[1], 2)}%")
+print()
+print("=" * 19)
+
+# data sets for entry into the graphs
+states = list(set(state_list))
+sales = list(state_dict.keys())
+
```
Comment (gagebroberg) [inline project_work.py:187]:
```suggestion

colors = ['b', 'g', 'r', 'c', 'm']
```

File: project_work.py [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+
+
+# creates the bar chart for question 2
+fig = plt.figure(1)
+plt.bar(states, sales)
```
Comment (gagebroberg) [inline project_work.py]:
```suggestion
plt.bar(states, sales, color= colors)

```This will add colors to the bar graph so it isn't as boring and is cool like Brandon's graphs.

Comment (gagebroberg) [inline project_work.py]:
```suggestion
plt.bar(states, sales, color= colors)

```
This will add colors to the bar graph so it isn't as boring and is cool like Brandon's graphs.

**Review by gagebroberg — CHANGES_REQUESTED** [review]
Except for the few comments I have left, I think this is looking pretty good. Good job Brandon on making some sick graphs mate.
